src/presentation/http/router/noteList.test.ts

```diff
@@ -1,27 +1,40 @@
-import userSessions from '@tests/test-data/user-sessions.json';
+import type User from '@domain/entities/user';
```

Comment:
seems redundant

Reply:
It is necessary for 5 line - I can't not define the type of the variable